CommMicro.py

```diff
@@ -325,49 +351,49 @@ def getDataBack(self,fname,tPlot,bPlot):
             cavDataList = FFt_math.parseCavDat(dFDat)
 
             # figure out cavities from filename for legend
-            fnameParts=fname.split('_')
+            fnameParts = fname.split('_')
             # find the fnamePart that starts with cav
             for part in fnameParts:
                 if part.startswith('cav'):
-                    cavnums=str(part[3:])
+                    cavnums = str(part[3:])
 
             tPlot.axes.cla()
             bPlot.axes.cla()
-            leGend=[]
-            leGend2=[]
-
-            for idx,cavData in enumerate(cavDataList):
-                if len(cavData)>0:
-                    leGend.append('Cav'+cavnums[idx])
-                    tPlot.axes.hist(cavData, bins=140,  
-                               histtype='step', log='True')
-               
-                    leGend2.append('Cav'+cavnums[idx])
-                    self.FFTPlot(bPlot,cavData)
-                     
+            leGend = []

```

Comment:
I think legend is for the top plot and legend2 is the bottom. The capital G is leftover from Bob.

Reply:
In which case I propose renaming to legendTop and legendBottom lol

Or better yet, a more descriptive name that describes what's actually in the plot

Reply:
I'm way ahead of you. In the other update I'm working on that adds more plots, these have more descriptive names. Will leave it as is here, since they will be replaced anyway.